_specifications/psp/0.1/general/initialize.md

```diff
@@ -0,0 +1,168 @@
+#### <a href="#initialize" name="initialize" class="anchor">Initialize Request (:leftwards_arrow_with_hook:)</a>
+
+The initialize request follows the same rules as the initialize request from the [LSP specification](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#initialize). Additions to this request are the new capabilities, and the subscribed events: A server can choose to dynamically or at startup subscribe to PSP/LSP events. The client will only send the subscribed events to the server.
+
```

Comment:
Since this command appears so early in the document, I'd suggest making repeated use of `server (plugin)` / `client (editor)` to make sure that the reader understands which is which and doesn't get confused. Though, it should probably not be done over the entire file, mentioning it this early on will help acclimate them to it.